### PR TITLE
Update documentation

### LINKED ISSUE #49: Update Documentation
## Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [ ] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

## Context

Documentation Issue.

## Expected Behavior

Documentation should be correct

## Current Behavior

There were recent changes in `Converter` parameters, documentation should reflect examples of consuming new parameter.

### PR DESCRIPTION
**Pull request by abhishek-jain-1999**
## Description

Now documentation example for consuming parameter ratio_ad_dp in QucikStart
And Readme has example and documentation now 

Fixes #49 

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Unit Tests
- [X] PEP 8 Formatting Validation

### CODE REVIEW

**Comment by abhishek-jain-1999** [comment]
@srgothi92 @rhdolin Please suggested me, if there is any more changes that needs to be done or if there is something I have done wrong

File: docs/QUICKSTART.rst [hunk 1]
```diff
@@ -45,6 +45,18 @@ More Example to instantiate Converter
 
        vcf2fhir.Converter('vcftests.vcf','GRCh37', 'aabc')
 
+-  Converts all variants in VCF. FHIR report assign homoplasmic vs.
```
Comment (srgothi92) [inline docs/QUICKSTART.rst:48]: This is good.

**Review by srgothi92 — COMMENTED** [review]
I don't have much thoughts on what can be added to documentation since I know how the library works. But, if you feel anything is missing please feel free to raise a PR for it.

File: README.md [hunk 2]
```diff
@@ -48,7 +117,41 @@ You can use python standard [logging](https://docs.python.org/3/library/logging.
 
 ### Documentation
 
-You can find the detailed documantation of the package on the official website [here](https://vcf2fhir.readthedocs.io/en/latest/).
+> ##### Class vcf2fhir.Converter ( vcf_filename = None, ref_build = None, patient_id = None, has_tabix = False, conv_region_filename = None, conv_region_dict = None, region_studied_filename = None, nocall_filename = None,ratio_ad_dp  =  0.99)
```
[the remark this answers is not in the document]
Comment (abhishek-jain-1999) [reply README.md:120]: so should I do something like this given below or remove it?

https://github.com/elimuinformatics/vcf2fhir/blob/d4f0f88c53f014af34f83d55574a42327b3be411/vcf2fhir/converter.py#L10-L90

File: README.md [hunk 3]
```diff
@@ -22,6 +22,75 @@ pip install vcf2fhir
 >>> vcf_fhir_converter = vcf2fhir.Converter('sample.vcf', 'GRCh37')
 >>> vcf_fhir_converter.convert()
 ```
+More Example to instantiate Converter
```
[the remark this answers is not in the document]
Comment (abhishek-jain-1999) [reply README.md:25]: okay I will remove it

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (abhishek-jain-1999) [reply README.md:120]: or maybe shift it to API.rst file.

File: docs/QUICKSTART.rst (hunk 1 above)
Comment (abhishek-jain-1999) [reply docs/QUICKSTART.rst:48]: Thank you 😃

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (srgothi92) [reply README.md:120]: API.rst file generates documentation from the code itself. Any API documentation should be added in code using `docstring` and we should wire up `API.rst` to generate documentation from docstring.

**Comment by srgothi92** [comment]
Can you squash all commits to one and add a nice commit message.

**Comment by abhishek-jain-1999** [comment]
Ya working on that

**Review by srgothi92 — APPROVED** [review]

**Pull request merged**